modules/assim.batch/R/pda.define.llik.R

```diff
@@ -88,17 +91,16 @@ pda.calc.error <-function(settings, con, model_out, run.id, inputs, bias.terms){
         SS[b] <- sum((bias.terms[bc,][b] * model_out[[k]] - inputs[[k]]$obs)^2, na.rm = TRUE)
       }
       
-      bc              <- bc + 1
+      bc <- bc + 1
       pda.errors[[k]] <- SS 
       SSdb[[k]]       <- log(SS)
       
     } else { # Gaussian
       
       SS <- sum((model_out[[k]] - inputs[[k]]$obs)^2, na.rm = TRUE)
       
-      pda.errors[[k]] <- SS 
-      SSdb[[k]]       <- log(SS)
```

Comment:
In the current context, I'm not 100% following what these two variables are, so I'm curious why they were deleted here but kept for the other cases